RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSequence_cff.py

```diff
@@ -57,19 +57,23 @@
 # RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECALUncorrected_cfi.py
 # RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cff.py
 # (particleFlowClusterECAL_cfi is generated automatically)
-from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSuperClusters_cff import lowPtGsfElectronSuperClusters
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSuperClusters_cfi import lowPtGsfElectronSuperClusters
 
 # Low pT electron cores
-from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronCores_cff import *
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronCores_cfi import lowPtGsfElectronCores
```

Comment:
I think that it's better to keep this and the toModify in a lowPtGsfElectronCores_cff so that the modifiers are the closest to their module defs.
This need for modifiers and lack of support for them in the fillDescriptions is a somewhat minor reason to keep defaultLowPtGsfElectronCores for the system-generated cfi and reduce possible confusion for proper consistent include of lowPtGsfElectronCores from a single place lowPtGsfElectronCores_cff (if it's ever needed more than just here).